baseplate/observers/prometheus.py

```diff
@@ -57,9 +58,6 @@ def __init__(self) -> None:
     def on_set_tag(self, key: str, value: Any) -> None:
         self.tags[key] = value
 
-    def get_prefix(self) -> str:
-        return f"{self.protocol}_server"
-
```

Comment:
Why is this method being removed? It would be nice for this to be mentioned in the commit message.

Reply:
Looks like it was removed because it was only used in tests. That said, the `$proto_$actor` syntax is used a bunch in the gauges that are set up, I would actually have preferred to see this used in those gauge definitions (as a static class method that's used in generating the string)

Reply:
Yeah it was just not used significantly anywhere.

I think Nathan is half right. I think originally that's how it was used, but then it was never fully developed and now it's just there to be tested but doesn't bring any value.

Reply:
To be clear I'm asking for `half used` => `actually used` instead of removing it.

Reply:
Will do this in a follow up PR to keep the PR focused on a single change.